eng/scripts/aggregate_javadoc_configuration.txt

```diff
@@ -0,0 +1,94 @@
+Group;Azure Core - Test;com.azure.core.test*
+Group;Azure Core - Tracing - OpenTelemetry;com.azure.core.tracing.opentelemetry*
+Group;Azure Agrifood;com.azure.verticals.agrifood.farming*
+Group;Azure Anomaly Detector;com.azure.ai.anomalydetector*
+Group;Azure App Configuration;com.azure.data.appconfiguration*
+Group;Azure Communication;com.azure.communication*
+Group;Azure Confidential Ledger;com.azure.security.confidentialledger*
+Group;Azure Container Registry;com.azure.containers.containerregistry*
+Group;Azure Cosmos;com.azure.cosmos*
+Group;Azure Device Update for IoT Hub;com.azure.iot.deviceupdate*
+Group;Azure Digital Twins;com.azure.digitaltwins.core*
+Group;Azure Document Translator;com.azure.ai.documenttranslator*
+Group;Azure Event Hubs;com.azure.messaging.eventhubs*
+Group;Azure Form Recognizer;com.azure.ai.formrecognizer*
+Group;Azure Identity;com.azure.identity*
+Group;Azure IoT Models Repository;com.azure.iot.modelsrepository*
+Group;Azure Key Vault;com.azure.security.keyvault*
+Group;Azure Mixed Reality Authentication;com.azure.mixedreality.authentication*
+Group;Azure Mixed Reality Remote Rendering;com.azure.mixedreality.remoterendering*
+Group;Azure Monitor - Ingestion;com.azure.monitor.ingestion*
+Group;Azure Monitor - Logs and Metrics query;com.azure.monitor.query*
+Group;Azure Monitor - OpenTelemetry Exporter;com.azure.monitor.opentelemetry.exporter*
+Group;Azure Personalizer;com.azure.ai.personalizer*
+Group;Azure Purview Catalog;com.azure.analytics.purview.catalog*
+Group;Azure Purview Scanning;com.azure.analytics.purview.scanning*
+Group;Azure Quantum Jobs;com.azure.quantum.jobs*
+Group;Azure Search Documents;com.azure.search.documents*
+Group;Azure Security Attestation;com.azure.security.attestation*
+Group;Azure Service Bus;com.azure.messaging.servicebus*
+Group;Azure Storage - Common;com.azure.storage.common*
+Group;Azure Storage - Blobs;com.azure.storage.blob*
+Group;Azure Storage Blob - Batch;com.azure.storage.blob.batch*
+Group;Azure Storage Blob - Changefeed;com.azure.storage.blob.changefeed*
+Group;Azure Storage Blobs - Cryptography;com.azure.storage.blob.cryptography*
+Group;Azure Storage Blobs - NIO;com.azure.storage.blob.nio*
+Group;Azure Storage Files - Share;com.azure.storage.file.share*
+Group;Azure Storage Files - Data Lake;com.azure.storage.file.datalake*
+Group;Azure Storage - Queues;com.azure.storage.queue*
+Group;Azure Text Analytics;com.azure.ai.textanalytics*
+Group;Azure Web Pub Sub;com.azure.messaging.webpubsub*
+
+# Adds the following links to external javadocs
+# Link;URL 
+Link;https://docs.oracle.com/javase/8/docs/api/
+Link;https://projectreactor.io/docs/core/release/api/
+Link;https://projectreactor.io/docs/netty/release/api/
+Link;https://fasterxml.github.io/jackson-annotations/javadoc/2.12/
+Link;https://fasterxml.github.io/jackson-core/javadoc/2.12/
+Link;https://fasterxml.github.io/jackson-databind/javadoc/2.12/
```

Comment:
Need to be updated to 2.13 but this isn't an issue for this PR, just a reminder for ourselves

Reply:
Will have a follow-up PR for updating the version.